src/App.js

```diff
@@ -18,8 +20,9 @@ class App extends Component {
             <MainNavbar />
             <Route exact path="/" component={MainPage} />
             <Route exact path="/user/:username" component={UserPanel} />
-            <Route exact path="/:slug" component={BlogLovWeb} />
+            <Route exact path="/:addblog/slug" component={BlogLovWeb} />
```

Comment:
Why? Its nonsense. We don't have variable 'addblog'. Nowhere. In this form routing our app is broken.